AppFramework/Event/GREYTouchInjector.m

```diff
@@ -318,4 +325,23 @@ inline static uint64_t getMachOTimeFromSeconds(CFTimeInterval seconds) {
   return time;
 }
 
+/**
+ * For iOS 14+, sets on the first _firstTouchForView property on the touchFlags struct of a UITouch.
+ *
+ * @param touch The UITouch being updated.
+ */
+static inline void SetTouchFlagPropertyInUITouch(UITouch *touch) {
+  typedef UITouchFlags (*UITouchFlagsGetVariableFunction)(id object, Ivar ivar);
+  UITouchFlagsGetVariableFunction getTouchFlagsFunction =
+      (UITouchFlagsGetVariableFunction)object_getIvar;
+  Ivar touchflags = class_getInstanceVariable([UITouch class], "_touchFlags");
+
+  UITouchFlags flags = getTouchFlagsFunction(touch, touchflags);
```

Comment:
The code I presented in https://github.com/google/EarlGrey/issues/1357#issuecomment-648921968 is more robust when future flags are added to the touch struct, as it ignores the size completely and only flips the first bit.